libcaf_io/src/ip_endpoint.cpp

```diff
@@ -36,6 +36,10 @@
 # include <netinet/ip.h>
 #endif
 
+#ifdef CAF_WINDOWS
+using sa_family_t = unsigned int;
```

Comment:
Microsoft uses a `short` for the `sin_family` and `sin6_family` fields: https://msdn.microsoft.com/en-us/library/windows/desktop/ms740496(v=vs.85).aspx

Reply:
Done